modules/programs/alacritty.nix

```diff
@@ -92,10 +92,12 @@ in
 
         theme = "${alacrittyTheme}/share/alacritty-theme/${cfg.theme}.toml";
       in
-      lib.mkIf (cfg.theme != null) {
-        general.import = lib.mkIf (lib.versionAtLeast cfg.package.version "0.14") [ theme ];
-        import = lib.mkIf (lib.versionOlder cfg.package.version "0.14") [ theme ];
-      };
+      lib.mkIf (cfg.theme != null) (
```

Comment:
Could you update tests to verify the expected behavior of the change. So we can see the expected toml for pre 0.14 and post 0.14. 

Also looks like a simpler solution would have been to include the `import` key in the guarded part no? We keep `mkIf` benefits and just don't create any of the key/values unless version matches.

Reply:
hi @khaneliman I will try to learn how to do the thing you suggested and also how to create a test for this. I have less than rudimentary understanding of Nix so it might take me a long time to find time and figure it out. I am happy to take any pointers/links to examples but have no expectations in having my hand held. Thanks for the suggestions.

Reply:
@khaneliman maybe I should be using `lib.mkMerge` to preserve the benefits of `lib.mkIf`? If that's not the right path, please let me know. I'll learn about updating the tests and give it a go. I can verify now that it will generate only one attribute based on the version number specified, which is the bug being fixed. Thanks again for suggestion.